Error building master  (...__m128 ) on Arch Linux

I run the following:

```
#$ git checkout master
#$ git pull --recurse-submodules ; git submodule init ; git submodule update
#$ git branch
  darktable-3.2.x
* master
```
then:

```
#$ rm -rf build/*
#$ ./build.sh --prefix /usr --buildtype Release

```
I get the error:

```
darktable/src/common/guided_filter.h:83:15: error: unknown type name ‘__m128’
   83 | static inline __m128 Kahan_sum_sse(const __m128 m, __m128 *c, const __m128 add)
      |               ^~~~~~
darktable/src/common/guided_filter.h:83:42: error: unknown type name ‘__m128’
   83 | static inline __m128 Kahan_sum_sse(const __m128 m, __m128 *c, const __m128 add)
      |                                          ^~~~~~
darktable/src/common/guided_filter.h:83:52: error: unknown type name ‘__m128’; did you mean ‘__int128’?
   83 | static inline __m128 Kahan_sum_sse(const __m128 m, __m128 *c, const __m128 add)
      |                                                    ^~~~~~
      |                                                    __int128
darktable/src/common/guided_filter.h:83:69: error: unknown type name ‘__m128’
   83 | static inline __m128 Kahan_sum_sse(const __m128 m, __m128 *c, const __m128 add)
      |                                                                     ^~~~~~
make[2]: *** [src/CMakeFiles/lib_darktable.dir/build.make:439: src/CMakeFiles/lib_darktable.dir/common/guided_filter.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:2344: src/CMakeFiles/lib_darktable.dir/all] Error 2
make: *** [Makefile:171: all] Error 2
```

I'm on Arch Linux. All the dependencies seems ok.

```
#$ gcc -v
Using built-in specs.
COLLECT_GCC=gcc
COLLECT_LTO_WRAPPER=/usr/lib/gcc/x86_64-pc-linux-gnu/10.2.0/lto-wrapper
Target: x86_64-pc-linux-gnu
Configured with: /build/gcc/src/gcc/configure --prefix=/usr --libdir=/usr/lib --libexecdir=/usr/lib --mandir=/usr/share/man --infodir=/usr/share/info --with-bugurl=https://bugs.archlinux.org/ --enable-languages=c,c++,ada,fortran,go,lto,objc,obj-c++,d --with-isl --with-linker-hash-style=gnu --with-system-zlib --enable-__cxa_atexit --enable-cet=auto --enable-checking=release --enable-clocale=gnu --enable-default-pie --enable-default-ssp --enable-gnu-indirect-function --enable-gnu-unique-object --enable-install-libiberty --enable-linker-build-id --enable-lto --enable-multilib --enable-plugin --enable-shared --enable-threads=posix --disable-libssp --disable-libstdcxx-pch --disable-libunwind-exceptions --disable-werror gdc_include_dir=/usr/include/dlang/gdc
Thread model: posix
Supported LTO compression algorithms: zlib zstd
gcc version 10.2.0 (GCC) 

```

The "darktable-3.2.x" version ( #$ git checkout darktable-3.2.x ) compiles without errors.

Thanks

--- a/src/common/guided_filter.h
+++ b/src/common/guided_filter.h
@@ -18,6 +18,8 @@
 
 #pragma once
 
+#include <xmmintrin.h>
+
 #include "common/darktable.h"
 #include "common/opencl.h"
 

